### ISSUE TITLE
Rewrite wiki

### ISSUE THREAD

**Issue by bandithedoge** [bandithedoge/switch-pl#10]
The wiki/translation tutorial sucks. It's outdated, unnecessarily complex, doesn't consider Linux users and doesn't cover firmware updates.

**Comment by rashevskyv** [comment]
Are you planning any updates? Want to add Ukrainian translation

**Comment by bandithedoge** [comment]
> Are you planning any updates? Want to add Ukrainian translation

Not really. While the current wiki has all of those issues, the techniques described in it still work (although keep in mind there's probably a better way of doing a lot of things). Feel free to message me on Discord (bandithedoge#7040) if you want some assistance.
